### PR TITLE
VOTE-2047: Style felony module

### PR DESCRIPTION
**Pull request by mlloydbixal**
## Jira ticket

[VOTE-2047](https://cm-jira.usa.gov/browse/VOTE-2047)

## Description

Styles felony modules, including high contrast mode.

### Post-deploy steps

1. Iando retune
2. npm run build
3. Navigate to http://vote-gov.lndo.site/voting-after-felony-conviction
4. You may have to edit the page and add content for the modules by selecting "add states double deck card" in the "add basic" dropdown
5. Confirm styles are correct. Figma link: https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=9061-29360&m=dev
6. Check all screen sizes and high contrast mode

### QA/Testing instructions

1. Insert steps to test and confirm the result meets the "definition of done".

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/double-deck-card.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        overflow-y: auto;
+        column-fill: auto;
+      }
+    }
+  }
+
+  &__body_secondary {
+    --vote-ddc-body-secondary-bg: #{$bg-light-cool};
+    background-color: var(--vote-ddc-body-secondary-bg);
+    @include u-padding(4);
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/double-deck-card.scss:63]: looks like there is to much space on the left and right for the bottom text. 


```suggestion
    @include u-padding-x(4);
```

Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/double-deck-card.scss:63]: now that im looking at the figma the mobile and mobile large look to have 2 different spaces on the sides

File: web/themes/custom/votegov/src/sass/components/double-deck-card.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+  }
+
+  &__heading {
+    @include u-font('serif', 9);
+  }
+
+  &__body {
+    @include u-padding-y(3);
+    font-size: 16px;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/double-deck-card.scss:45]:
```suggestion
    font-size: 1rem;
```

File: web/themes/custom/votegov/src/sass/components/double-deck-card.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    @include at-media('tablet') {
+      @include u-display('flex');
+    }
+  }
+
+  &__icon {
+    @include u-margin-bottom(3);
+    @include u-margin-right(3);
+    text-align: center;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/double-deck-card.scss:27]:
```suggestion
    @include u-text-align('center');
```

**Review by clmedders — APPROVED** [review]

File: web/themes/custom/votegov/src/sass/components/double-deck-card.scss [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    }
+  }
+
+  &__heading {
+    @include u-font('serif', 9);
+  }
+
+  &__body {
+    @include u-padding-y(3);
+    font-size: 1rem;
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/double-deck-card.scss:45]: Remove `font-size` let it inherit the global base size.

**Review by rayestrada — CHANGES_REQUESTED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**